test/cluster/test_topology_upgrade_stuck.py

```diff
@@ -15,12 +15,16 @@
 from test.pylib.manager_client import ManagerClient
 from test.pylib.scylla_cluster import gather_safely
 from test.pylib.util import wait_for_cql_and_get_hosts, wait_for_first_completed
-from test.cluster.conftest import skip_mode
 from test.cluster.util import reconnect_driver, enter_recovery_state, \
         delete_raft_data_and_upgrade_state, log_run_time, wait_until_upgrade_finishes as wait_until_schema_upgrade_finishes, \
         wait_until_topology_upgrade_finishes, delete_raft_topology_state, wait_for_cdc_generations_publishing, \
         check_system_topology_and_cdc_generations_v3_consistency
 
+pytestmark = [
```

Comment:
You are not supporting args 

last SI run failed because of this 

```
pytestmark = [
    pytest.mark.skip_mode(mode='release', reason='error injections are not supported in release mode'),
    pytest.mark.skip_mode(mode='debug', reason='test performs many topology changes')
]
```